plugins/kernel/pyparts/memory.py

```diff
@@ -9,10 +10,39 @@
     VMSTAT,
     BUDDY_INFO,
     SLABINFO,
+    KernelChecksBase
 )
 
 YAML_PRIORITY = 1
 
+class KernelOOMChecks(KernelChecksBase, checks.EventChecksBase):
+
+    def __init__(self):
+        super().__init__(yaml_defs_label='memory-check')
+
+    def process_results(self, results):
+        """ See defs/events.yaml for definitions. """
+        info = {}
+        for defs in self.event_definitions.values():
+            for label in defs:
+                _results = results.find_by_tag(label)
+                if label == "oom":
+                    if _results:
+                        first_process_oomd = _results[0].get(3)
+                        time_oomd = _results[0].get(1) + " " +  _results[0].get(2)
+                        msg = ("oom-killer had been invoked, first for process ")
+                        issue = issue_types.MemoryWarning(msg + first_process_oomd + " at " + time_oomd);
+                        issues_utils.add_issue(issue);
+                        info['oom']=time_oomd
```

Comment:
can we rename "oom"  to something like "oom-event"

Reply:
ack